server/config/schemas/unter.sql

```diff
@@ -0,0 +1,71 @@
+DROP DATABASE IF EXISTS Unter;
+CREATE DATABASE Unter;
+USE Unter; 
+
+DROP TABLE IF EXISTS DRIVER; 
+CREATE TABLE DRIVER(
+	fname varchar(25) NOT NULL, 
+	lame varchar(25) NOT NULL, 
+	dob date NOT NULL, 
+	userID int NOT NULL, 
+	totalTrips int NOT NULL DEFAULT 0,
+	rating int NOT NULL DEFAULT 0,
+	CONSTRAINT pk_driver primary key(userID) 
+);
+
+DROP TABLE IF EXISTS PASSENGER; 
+CREATE TABLE PASSENGER(
+	fname varchar(25) NOT NULL, 
+	lame varchar(25) NOT NULL, 
+	dob date NOT NULL, 
+	userID int NOT NULL, 
+	totalTrips int NOT NULL DEFAULT 0,
+	rating int NOT NULL DEFAULT 0,
+	CONSTRAINT pk_passenger primary key(userID)
+);
+
+DROP TABLE IF EXISTS TRIP; 
+CREATE TABLE TRIP(
```

Comment:
May need to be updated since Driver - Trip is 1/n not n/n. Same with Passenger - Trip.